appveyor.yml

```diff
@@ -4,7 +4,7 @@ environment:
   global:
     SALT_FILEROOT: '%APPVEYOR_BUILD_FOLDER%'
     SALT_STATE: name-computer
-    SALT_URL: 'https://repo.saltstack.com/windows/Salt-Minion-2018.3.4-Py2-AMD64-Setup.exe'
+    SALT_URL: https://repo.saltproject.io/windows/Salt-Minion-3004.2-1-Py3-AMD64-Setup.exe
```

Comment:
So, when we move to 3005.1+ (so that this formula works for EL8), this value will need to be updated?

Reply:
Indeed. We don't yet have a way of letting dependabot handle these version updates for us.